packages/element/src/create-interpolate-element.js

```diff
@@ -0,0 +1,232 @@
+/**
+ * Internal dependencies
+ */
+import { getTokenCount, resetTokenCount } from './token-count';
+import { createElement, Fragment, isValidElement } from './react';
+
+/**
+ * External dependencies
+ */
+import { escapeRegExp, flatMap } from 'lodash';
+
+const getHasPropValue = ( config ) => !! config.value;
+const getHasChildren = ( config ) => !! config.hasChildren;
+
+/**
+ * Transforms a given string value into a collection of html like tags.
+ *
+ * @example
+ *
+ * If you provide `something` as the tagString, this will return:
+ *
+ * ```js
+ * const tags = getTagsFromString( 'something' );
+ * const tagsAre = tags === [
+ *     '<something>',
+ *     '</something>',
+ *     '<something/>',
+ * ];
+ * ```
+ *
+ * @param {string} tagString  Something like `a%1`.
+ *
+ * @return {string[]} The generated tags for the given string.
+ */
+export const getTagsFromString = ( tagString ) => {
```

Comment:
`getTagsFromString` method is never exposed to the public, does it need to be exported? I don't see unit tests for it, either. Not that it needs to be unit tested, but I'm trying to understand why it's exposed.

Reply:
export was accidental from the extraction (moved in from another file).

Reply:
removed export in a02fcf7